### DISCUSSION TITLE
Does this project have an API? I mean, is it possible to pass my text via API and receive the response back?

### DISCUSSION THREAD

**Discussion by palomaarce** [xtekky/gpt4free#327] [Q&A]
Does this project have an API? I mean, is it possible to pass my text via API and receive the response back?

**Comment by sebramirez44** [comment]
I'm wondering the same thing, I would assume you make a request to the localhost port the docker thing is running from, but I have no clue what request would be needed

**Comment by xiangsx** [comment]
> Does this project have an API? I mean, is it possible to pass my text via API and receive the response back?

you can find gpt4free-ts in readme.That provide api
